Makefile

```diff
@@ -58,7 +58,7 @@ run_staging_db:
 	$(staging_compose_cmd) run --rm staging_db postgres -V
 
 ## Migrate databases and create shapefiles to synchronize with static_files
-build_migrate_staging: run_staging_db
+migrate_staging: run_staging_db
```

Comment:
It would be nice to put migrate_staging in the deploy_staging:
```
deploy_staging: migrate_staging
```

Reply:
Thanks for contributing @xmnlab!

Reply:
Thanks @xmnlab !